ugbook-back/src/schemas/comment.schema.ts

```diff
@@ -0,0 +1,16 @@
+import { Prop, Schema, SchemaFactory, unique } from '@nestjs/mongoose';
+import { Document } from 'mongoose';
+
+export type CommentDocument = Comment & Document;
+
+@Schema()
+export class Comment {
+    
+    @Prop({ required: true })
+    @unique()
+    authorId: string;
```

Comment:
authorId can be a string type but I think objectId would be a better choice.

Reply:
Corrected.